scripts/upload_config.sh

```diff
@@ -33,11 +34,11 @@ SRCPATH=${SCRIPTPATH}/..
 export CHANNEL=$2
 export FULLVERSION=$($SRCPATH/scripts/compute_build_number.sh -f)
 
-TEMPDIR=$(mktemp -d 2>/dev/null || mktemp -d -t "tmp")
+TEMPDIR=$(mktemp -d -t "upload_config.tmp.XXXXXX")
```

Comment:
I was finding that when storage on my instance was full, this first call was silently failing, and then the second call would *also* fail, but not because of storage. All mktemp formats need a section of "XXX" to decide where to put the random ID.

The script was failing with an error of `not enough Xs in template "tmp"` when the true error was out of disk. Very confusing.

Solution here is to not even try to use a default tmp format, but instead supply our own. I added the string "upload_config" to the front to make them easily identifiable and removable